packages/nx-extensions/src/executors/assert-built-esm-and-cjs/executor.ts

```diff
@@ -0,0 +1,57 @@
+import { ExecutorContext } from '@nrwl/devkit';
+import { spawnSync } from 'child_process';
+import { existsSync, mkdirSync, writeFileSync } from 'fs';
+import * as path from 'path';
+import { AssertBuiltEsmAndCjsExecutorSchema } from './schema';
+
+/**
+ * Test whether a module can be imported as both ESM and CJS.
+ *
+ * @param options
+ * @param context
+ * @returns
+ */
+export default async function runExecutor(
+	options: AssertBuiltEsmAndCjsExecutorSchema,
+	context: ExecutorContext
+) {
+	const buildDir = options.outputPath.split('/')[0];
+	const testsPath = path.join(context.root, buildDir, 'test-esm-cjs');
+	if (!existsSync(testsPath)) {
+		mkdirSync(testsPath);
```

Comment:
if we pass the `{recursive: true}` option to this then I believe we never need the `exists` check, because the function won't fail. it will simply create the directory as long as it doesn't already exist

Reply:
❤️